packages/wrangler/src/cert/cert.ts

```diff
@@ -0,0 +1,203 @@
+        );
+        const certResponse = await uploadCaCertificateFromFs(accountId, {
+            certificates: caCert,
+            ca: true,
+            name: name,
+        });
+        const expiresOn = new Date(certResponse.expires_on).toLocaleDateString();
+        logger.log(
+            name
+                ? `Success! Uploaded CA Certificate ${name}`
+                : `Success! Uploaded CA Certificate`
+        );
+        logger.log(`ID: ${certResponse.id}`);
+        logger.log(`Issuer: ${certResponse.issuer}`);
+        logger.log(`Expires on ${expiresOn}`);
+    }
+});
+
+// wrangler cert list
+export const certListCommand = createCommand({
+	metadata: {
+		description: "List uploaded mTLS certificates",
+		status: "stable",
+        owner: "Product: SSL"
+	},
+	async handler(_, { config }) {
+        const accountId = await requireAuth(config);
+        const certificates = await listMTlsCertificatesCA(accountId, {});
+        for (const certificate of certificates) {
+            logger.log(`ID: ${certificate.id}`);
+            if (certificate.name) {
+                logger.log(`Name: ${certificate.name}`);
+            }
+            logger.log(`Issuer: ${certificate.issuer}`);
+            logger.log(
+                `Created on: ${new Date(certificate.uploaded_on).toLocaleDateString()}`
+            );
+            logger.log(
+                `Expires on: ${new Date(certificate.expires_on).toLocaleDateString()}`
+            );
+            if (certificate.ca) {
+                logger.log(
+                    `CA: ${certificate.ca}`
+                )
+            }
+            logger.log("\n");
+        }
+    }
+});
+
+// wrangler cert delete
+export const certDeleteCommand = createCommand({
+	metadata: {
+		description: "Delete an mTLS certificate",
+		status: "stable",
+        owner: "Product: SSL"
+	},
+    args: {
+        id: {
+            description: "The id of the mTLS certificate to delete",
+			type: "string"
+        },
+        name: {
+            description: "The name of the mTLS certificate record to delete",
+			type: "string"
+        }
+    },
+	async handler({id, name}, { config }) {
+        const accountId = await requireAuth(config);
+        if (id && name) {
+            return logger.error(`Error: can't provide both --id and --name.`);
+        } else if (!id && !name) {
+            return logger.error(`Error: must provide --id or --name.`);
+        }
+        let certificate: MTlsCertificateResponse;
+        if (id) {
+            certificate = await getMTlsCertificate(accountId, id);
+        } else {
+            certificate = await getMTlsCertificateByName(accountId, name as string);
+        }
+
+        const response = await confirm(
+            certificate.name
+                ? `Are you sure you want to delete certificate ${certificate.id} (${certificate.name})?`
+                : `Are you sure you want to delete certificate ${certificate.id}?`
+        );
```

Comment:
```suggestion
        const response = await confirm(
                `Are you sure you want to delete certificate ${certificate.id}${certificate.name ? `(${certificate.name})` : ""}?`
        );
```

nit 
(and for other similar log statements too)